### ISSUE TITLE
SVG are not fully accessible for Screen Reader User (Post Meta)

### ISSUE THREAD

**Issue by 00travelgirl00** [godaddy-wordpress/go#531]
**Describe the bug:**
When navigating with a screen reader, the svg-Icons are read out as "Image". That desicription has no value to screen reader users. It would be better to add an alt-text that describes the SVG or to hide the icon via Aria-Label (aria-hidden: true). 

aria-hidden: true should only be used, if the information of the icon is also available via text. 
That is the case for the svg for post author, post date, comments and featured post. Therefore the svg can be hidden for screen readers to reduce noice. 

**Screenshot(s):**
<img width="1269" alt="Bildschirmfoto 2020-06-08 um 16 31 57" src="https://user-images.githubusercontent.com/21318377/84043309-66a55d00-a9a6-11ea-9aac-dee61578db78.png">

Behavoir: (Video not listet) https://youtu.be/qV1dcRo2we4
(my Screen reader is in German. "Bild" means "Image" and has no value for screen reader users. 

**Priority** (select one):
- [ ] High - Functional blocker
- [ ] Medium - Doesn't work as expected 
- [x] Low - Not a big deal but we'd want to get to this eventually

More informations on this topic
https://www.456bereastreet.com/archive/201609/hiding_inline_svg_icons_from_screen_readers/

**Comment by 00travelgirl00** [comment]
Same applies for the social icons in the footer

**Comment by richtabor** [comment]
Hey @00travelgirl00, thanks for the report! I prepped a PR for this, mind taking a look and letting me know what you think?
